subt/docker/robotika/ros/robotika/CMakeLists.txt

```diff
@@ -0,0 +1,198 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(robotika)
+
```

Comment:
Can the commented out lines be deleted? Anything that makes this PR smaller counts.

Reply:
ok